code/parachain/runtime/composable/src/ibc.rs

```diff
@@ -163,8 +160,8 @@ impl pallet_ibc::Config for Runtime {
 	type ParaId = parachain_info::Pallet<Runtime>;
 	type RelayChain = RelayChainId;
 	type WeightInfo = weights::pallet_ibc::WeightInfo<Self>;
-	type AdminOrigin = EnsureRootOrOneThirdNativeTechnical;
-	type FreezeOrigin = EnsureRootOrOneThirdNativeTechnical;
+	type AdminOrigin = EnsureRootOrTwoThirdNativeTechnical;
+	type FreezeOrigin = EnsureRootOrTwoThirdNativeTechnical;
```

Comment:
btw freeze is less secure op and should be fast to do to stop hack. it should be 1/3